website/content/en/docs/getting-started-with-kops/_index.md

```diff
@@ -0,0 +1,195 @@
+
+---
+title: "Getting Started with Karpenter using kOps"
+linkTitle: "Getting Started with kOps"
+weight: 10
+---
+
+In this example, the cluster will be running on Amazon Web Services (AWS) managed by [kOps](https://kops.sigs.k8s.io/).
+Karpenter is designed to be cloud provider agnostic, but currently only supports AWS. Contributions are welcomed.
+
+Karpenter is supported on kOps as of 1.24.0-alpha.2, but sits behind a feature flag as the interface between kOps and Karpenter is
```

Comment:
I'm not sure about the long term viability of wrapping Karpenter's API with an instance group and the conflicting concerns between whether or not Karpenter or Kops owns the LT. Can we be ultra crisp about this and let customers know that they should only take a dependency on this alpha kops integration if they're prepared for significant incompatible changes?

Reply:
Emphasized it some more